contracts/HypERC20Collateral.sol

```diff
@@ -15,13 +19,19 @@ contract HypERC20Collateral is TokenRouter {
     using SafeERC20 for IERC20;
 
     IERC20 public immutable wrappedToken;
+    address public tokenAddress;
 
     /**
      * @notice Constructor
      * @param erc20 Address of the token to keep as collateral
      */
     constructor(address erc20) {
         wrappedToken = IERC20(erc20);
+        tokenAddress = erc20;
+    }
+
+    function delegateVotes(address _L1VoteDelegator) external onlyOwner{
```

Comment:
Actually thinking about this more, the typical approach to implement ERC20 supersets is to create another version, so I think you should do the same think here as with HypERC20 and create a `HypERC20CollateralVotes` contract that inherits from `HypERC20Collateral`

Reply:
Got it will create a new HypERC20CollateralVotes